pkg/utils/git/repo.go

```diff
@@ -73,3 +64,7 @@ func HasUpstreamChanges() (bool, error) {
 
 	return remote == ref.Hash().String(), nil
 }
+
+func Init() (string, error) {
+	return gitRaw("init")
```

Comment:
Why it's "init"?

Reply:
it's for test purposes `git init`. I need to create a git repo for some comands